docs/index.md

```diff
@@ -1,33 +1,54 @@
+Operator Lifecycle Manager (OLM) is an open-source [CNCF](https://www.cncf.io/) project with the mission to manage the 
+lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its is to make installing, 
+running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and PaaS administrators, 
+throughout the lifecycle of the underlying cluster.
 
-OLM has been helping to define lifecycles for these extensions in which the extensions
+OLM was focused on a particular type of cluster extension: [operators](https://operatorhub.io/what-is-an-operator#:~:text=is%20an%20Operator-,What%20is%20an%20Operator%20after%20all%3F,or%20automation%20software%20like%20Ansible.). 
+An operator is a Kubernetes application composed of one or more controllers and one, or more, API extensions, 
+or [CustomResourceDefinitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)) (CRDs).
 
-  * get installed, potentially causing other extensions to be installed as well as dependencies
-  * get customized with the help of customizable configuration at runtime 
-  * get upgraded to newer version/s following upgrade paths defined by extension developers 
-  * and finally, get decommissioned and removed.
+OLM helped define lifecycles for these extensions in which the extensions:
 
-In the dependency model, extensions can rely on each other for required services that are out of scope of the primary purpose of an extension, allowing each extension to focus on a specific purpose. 
+* get installed, potentially causing other dependency extensions to be installed as well
+* can be customized through configuration at runtime
+* get upgraded to newer version/s following upgrade paths defined by extension developers
+* and finally, get decommissioned and removed.
 
-OLM also prevents conflicting extensions from running on the cluster, either with conflicting dependency constraints or conflicts in ownership of services provided via APIs. Since cluster extensions need to be supported with an enterprise-grade product lifecycle, there has been a growing need for allowing extension authors to limit installation and upgrade of their extension by specifying additional environmental constraints as dependencies, primarily to align with what was tested by the extension author's QE processes. In other words, there is an evergrowing ask for OLM to allow the author to enforce these support limitations in the form of additional constraints specified by extension authors in their packaging for OLM.
+The first iteration of OLM, termed v0, included several concepts and features targeting the stability, security, and supportability of the life-cycled applications, for instance:
+* A dependency model that enabled cluster extensions to focus on their primary purpose by delegating out of scope behavior to dependencies.
```

Comment:
right, we are acknowledging that in V1 we are doing dependency resolution.  IMO we can just remove it from the first section.

Reply:
I've condensed the previous section to something a little more general and left this section as is.